### ISSUE TITLE
Label text is inacessible (WebAIM); opacity (contrast) and size

### ISSUE THREAD

**Issue by brylie** [mozilla/metrics-graphics#470]
Label text is styled in such a way so that it is hard to read. Essentially, the text is grey (black with 50% opacity) on a white background and small (0.8rems). While this may be done to clarify the graph, it makes the labels hard to read (and may actually be detrimental to interpreting the graph).

Basic calculation:
* black with 50% opacity = rgba(217, 216, 216, 0.99) according to chrome dev tools' color picker
* rgba(217, 216, 216, 0.99) = #d9d8d8 (as per [rgba to hex converter](http://jsfiddle.net/rymL991j/))
* #d9d8d8 foreground text on #fff background text fails all levels of [Web Content Accessibility Guidelines contrast check](http://webaim.org/resources/contrastchecker/)

Increasing font size and opacity will improve contrast and readability.

**Comment by almossawi** [comment]
Thank you. Targeting 2.7.

**Comment by hamilton** [comment]
:thumbsup: Thanks for the issue

**Comment by brylie** [comment]
Cheers. This is a great charting package! :smile:

**Comment by almossawi** [comment]
I've upped the opacity and size of the labels and lines. We probably won't be able to pass all the tests at that link without making things hella ugly, as the kids might say :) Thoughts? An alternative _accessibility_ stylesheet perhaps?

**Comment by hamilton** [comment]
I think we can do what we can. I think there are some optimizations that we can do to improve basic accessibility, and ultimately an alternate stylesheet that favors bigger labels and text is a great option.

Let's visit this on Thursday.

**Comment by almossawi** [comment]
I've added a new [accessibility stylesheet](https://github.com/mozilla/metrics-graphics/blob/master/examples/css/metricsgraphics-demo-accessible.css) and a new link to the top of the examples pages.
